examples/sample/opo/embed.conf

```diff
@@ -0,0 +1,245 @@
+json.detect.time.number.on = true
+json.detect.time.number.min = 1400000000
+json.detect.time.number.max = 1600000000
+json.detect.time.number.decimals = 9
+
+# JSON imports are converted to triples but not RDF compliant triples. In an
+# RDF triple the subject must be either a blank or a IRI. The predicate must
+# be a IRI. Since imported JSONs do not have namespaces associated with them a
+# default namespace is used to make the JSON literals into IRIs using a
+# default namespace.
+rdf.default.namespace = http://localhost#
+
+################################################################################
+# IRI aliases can be registered. These aliases will take precedence over any
+# prefix defined in a TURTLE import. Note that the commented out sample are
+# most likely out of date. Thats one of the problems with including the date
+# in the URL. It is also an advantage is referring to an older specification.
+ns.dct = http://purl.org/dc/terms/
+ns.foaf = http://xmlns.com/foaf/0.1/
+ns.gr = http://purl.org/goodrelations/v1#
+ns.org = http://www.w3.org/ns/org#
+ns.owl = http://www.w3.org/2002/07/owl#
+ns.prov = http://www.w3.org/ns/prov#
+ns.rdf = http://www.w3.org/1999/02/22-rdf-syntax-ns#
+ns.rdfs = http://www.w3.org/2000/01/rdf-schema#
+ns.skos = http://www.w3.org/2004/02/skos/core#
+ns.time = http://www.w3.org/2006/time#
+ns.vcard = http://www.w3.org/2006/vcard/ns#
+ns.xsd = http://www.w3.org/2001/XMLSchema#
+#ns.xsd = https://www.w3.org/TR/xmlschema11-2/#
+#ns.xsd = https://www.w3.org/TR/xmlschema-2/#
+
+# N-Triples and N-Quads literal strings can have a type associated with the
+# string. Opo attempts to convert the strings to native types if possible. To
+# do so it looks for know IRI prefixes followed by the type. The supported
+# types are 'integer', 'double', 'time', and 'datetime'. The listed
+# type.prefixes are used to form the IRIs that trigger the conversion of a
+# string to the respective native type. A comma separated list is expected.
+type.prefixes = http://www.w3.org/2001/XMLSchema#,http://www.w3.org/TR/xmlschema11-2/#,http://www.w3.org/TR/xmlschema-2/#
+
+# Multiple handlers can be running at the same time if using different
+# paths. The view conf.js would have to change to use one path or the other by
+# setting the wab.pathPrefix value.
+
+#handler.sample.path = /v1/**
+#handler.sample.max_out = 10
+#handler.sample.cmd = examples/sample/spawn.rb -I lib -t 4
+#handler.sample.cmd = spawn.rb -v
+#handler.sample.timeout = 5.0
+
+handler.embed.path = /v1/**
+# An embedded handler is denoted by a class value.
+handler.embed.class = SampleController
+
+################################################################################
+# When using opo-rub or other opo version with embedded interpreters the
+# 'embed' options can be used. The handler options are also extended to
+# support calling the embedded interpreter.
+
+embed.load_path = .,lib,examples/sample
```

Comment:
prefer a space after comma for readability:
```diff
- embed.load_path = .,lib,examples/sample
+ embed.load_path = ., lib, examples/sample
```
or preferably denoted as an Array with `[]`

Reply:
I'll make a note of that for OpO. Can't do anything about it here.